pkg/controller/plan/migration.go

```diff
@@ -675,6 +692,11 @@ func (r *Migration) execute(vm *plan.VMStatus) (err error) {
 			err = nil
 			break
 		}
+		err = r.kubevirt.DeletePVCConsumerPod(vm)
```

Comment:
why do we need to remove it here as well? isn't it enough to delete in the cleanup code?

Reply:
cleanup is called when restarting/archiving the plan. here it's - we done with the migration, let's clean it.

Reply:
oh, i thought we do based on the above discussion:

> > 1. Cleanup - do we wish to clean the pod even without archive the plan?

> it would be better, yes